### PR TITLE
[NUI] Make Copy, Cut and Paste APIs public

### PR DESCRIPTION
**Pull request by shrouqsabah**
Signed-off-by: Shrouq Sabah s.sabah@samsung.com
### Description of Change ###

Public open API list

- TextUtils.cs
    - public static string CopyToClipboard(TextEditor textEditor)//API
    - public static string CopyToClipboard(TextField textField)//API
    - public static string CutToClipboard(TextEditor textEditor)//API
    - public static string CutToClipboard(TextField textField)//API
    - public static void PasteTo(TextEditor textEditor)//API
    - public static void PasteTo(TextField textField)//API
    

- ACR: TCSACR-455
   https://code.sec.samsung.net/jira/browse/TCSACR-455

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs [hunk 1]
```diff
@@ -1819,8 +1817,7 @@ public static string CopyToClipboard(TextField textField)
         /// </summary>
         /// <param name="textEditor">The textEditor control from which the text is cut.</param>
         /// <returns>The cut text.</returns>
-        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs:1820]: the "... and ... and ..." doesn't sound right, maybe write it "Cut the previously selected text from the text control and **copy it** into the clipboard.", the returned value is described with returns tag and this is enough.

File: src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs [hunk 2]
```diff
@@ -1846,8 +1842,7 @@ public static string CutToClipboard(TextField textField)
         /// Paste the most recently copied/cut text from the clipboard and into the text control.
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs:1842]: Please describe how is it pasted. Such details can go into \<remarks\> tag.
What I'm interested in: does the pasted text replace the all the texteditor contents, or is it added at the end, replaces the selected text (that's what I'm used to in text editors on PC)?

Comment (shrouqsabah) [reply src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs]: @wiertel , 
Done.
Please could check and let me know what you see.

File: src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs (hunk 1 above)
Comment (shrouqsabah) [reply src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs:1820]: @wiertel , 
Done.

**Review by wiertel — APPROVED** [review]

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**